states/OH/governor.yaml

```diff
@@ -1,49 +1,183 @@
 contact_form:
   steps:
-    - visit: "http://apps.das.ohio.gov/governorcommunications/contactgovernor.aspx"
+    - visit: "https://governor.ohio.gov/wps/portal/gov/governor/contact"
+    - select:
+        - name: "select-issue"
+          selector: "#select-issue"
+          value: "Other"
```

Comment:
Hm, general would seem to work as well, though I'm not sure which would be better.

Reply:
I like General better, but we usually don't have that as an option.